packages/web-client/app/components/card-pay/issue-prepaid-card-workflow/index.ts

```diff
@@ -50,6 +52,18 @@ class IssuePrepaidCardWorkflow extends Workflow {
         new WorkflowCard({
           author: cardbot,
           componentName: 'card-pay/layer-two-connect-card',
+          onRender: () => {
+            let layer2Network = this.owner.lookup(
+              'service:layer2-network'
+            ) as Layer2Network;
+
+            layer2Network.waitForAccount.then(() => {
+              let MIN_DAI_AMOUNT = new BN(toWei('50'));
```

Comment:
Consider extracting this to a modue-scoped const.

Reply:
Yep, will get this from a shared configuration with the face value card.